var/spack/repos/builtin/packages/libxsmm/package.py

```diff
@@ -23,20 +23,40 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
 from spack import *
+from glob import glob
 
 
 class Libxsmm(Package):
-    '''LIBXSMM is a library for small dense and small sparse matrix-matrix
-    multiplications targeting Intel Architecture (x86).'''
+    '''Library targeting Intel Architecture
+    for small, dense or sparse matrix multiplications,
+    and small convolutions.'''
 
     homepage = 'https://github.com/hfp/libxsmm'
-    url      = 'https://github.com/hfp/libxsmm/archive/1.4.3.tar.gz'
```

Comment:
There is no need to update the url; Spack will automatically do the right thing given the version declarations.